oleggerasinovich/Titanic.ipynb

```diff
@@ -0,0 +1,1686 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            'Survived': 'survived_sum', 
</pre><p> </p><p>Actually, this is not a pure sum column (despite the method of summation), it is count. Cause you've counted the number of survived here (survived = 1, and it was summed to the number of the survived people).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/u2-de-students-lab/titanic_exploration/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>